### PR TITLE
Increase maximum reporting time and log reporting progress

### PR DESCRIPTION
**Pull request by Artem-Kuz**
Increased the maximum time for the queue to 10 minutes.
Signed-off-by: Artem Kuznetsov <artem.kuznetsov@testproject.io>

### CODE REVIEW

File: TestProject.OpenSDK/Internal/ReportsQueue.cs [hunk 1]
```diff
@@ -30,7 +31,12 @@ public class ReportsQueue
         /// <summary>
         /// Maximum amount of time to wait in milliseconds before forcibly terminating the queue.
         /// </summary>
-        private const int REPORTS_QUEUE_TIMEOUT = 10000;
+        private const int REPORTS_QUEUE_TIMEOUT = 10000 * 60;
```
Comment (vitalybu) [inline TestProject.OpenSDK/Internal/ReportsQueue.cs:34]: Why is this required?
Please provide more information in the PR description.
In any case, it's more readable to write this change as:
```c#
private const int REPORTS_QUEUE_TIMEOUT = 1000 * 60 * 10; // 10 minutes
```

>

**Review by vitalybu — CHANGES_REQUESTED** [review]

File: TestProject.OpenSDK/Internal/ReportsQueue.cs (hunk 1 above)
Comment (Artem-Kuz) [reply TestProject.OpenSDK/Internal/ReportsQueue.cs:34]: This is because when using a remote VDI, and lots of steps should be reported, it takes more than 10 seconds allowed for the reporting queue to finish.
https://dev.testproject.io/jira/browse/TP-15542

Comment (vitalybu) [reply TestProject.OpenSDK/Internal/ReportsQueue.cs]: I can see the value in updating the value from 10 seconds to 60 seconds (1 minute), but why 10 minutes?
Also, please make sure to not add internal links in public comments (removed one from your comment).

**Comment by s-glatshtein** [comment]
Closing this PR. We will open a separate task and solve this issue differently.
